test/yulPhaser/AlgorithmRunner.cpp

```diff
@@ -434,17 +434,21 @@ BOOST_FIXTURE_TEST_CASE(run_should_overwrite_existing_file_if_autosave_file_spec
 	BOOST_TEST(readLinesFromFile(m_autosavePath) != originalContent);
 }
 
-BOOST_FIXTURE_TEST_CASE(run_should_not_save_population_to_file_if_autosave_file_not_specified, AlgorithmRunnerAutosaveFixture)
-{
-	m_options.maxRounds = 5;
-	m_options.populationAutosaveFile = std::nullopt;
-	AlgorithmRunner runner(m_population, {}, m_options, m_output);
-	assert(!fs::exists(m_autosavePath));
-
-	runner.run(m_algorithm);
-
-	BOOST_TEST(!fs::exists(m_autosavePath));
-}
+// Disabled due to a fairly obscure bug in gcc13+
+// https://gcc.gnu.org/bugzilla/show_bug.cgi?id=109561
+// The issue is on line 444, and more specifically the Options struct std::optional<std::string>
+
+//BOOST_FIXTURE_TEST_CASE(run_should_not_save_population_to_file_if_autosave_file_not_specified, AlgorithmRunnerAutosaveFixture)
+//{
+//	m_options.maxRounds = 5;
+//	m_options.populationAutosaveFile = std::nullopt;
+//	AlgorithmRunner runner(m_population, {}, m_options, m_output);
+//	assert(!fs::exists(m_autosavePath));
+//
+//	runner.run(m_algorithm);
+//
+//	BOOST_TEST(!fs::exists(m_autosavePath));
+//}
```

Comment:
It would be better to use multi-line comments for things like this to avoid having non-indent tabs again, now that I fixed it in #15026.